admin_command.go

```diff
@@ -5,8 +5,8 @@ type AdminCommand struct {
 	Alias     string       `yaml:"alias" json:"alias"`
 	Timeout   uint         `yaml:"timeout" json:"timeout"`
 	RunType   string       `yaml:"run_type" json:"run_type"`
-	Component string       `yaml:"component" json:"component"`
-	Image     CommandImage `yaml:"image" json:"image"`
+	Component string       `yaml:"component" json:"component" validate:"componentexists"`
+	Image     CommandImage `yaml:"image" json:"image"` // TODO: validate exists
```

Comment:
this one is tough. i will have to write another validator. we really should try to collapse all these to use the same format.